host-controller/src/main/java/org/jboss/as/host/controller/logging/HostControllerLogger.java

```diff
@@ -1415,4 +1415,10 @@ void noDomainControllerConfigurationProvidedForAdminOnly(String policyAttribute,
     @Message(id = 210, value = "A slave host controller may not be added using add(). Please add a host, omitting this parameter, and configure the remote domain controller using write-attribute.")
     OperationFailedException cannotAddSlaveHostAfterBoot();
 
+    @Message(id = 211, value = "Boot configuration validation failed")
+    OperationFailedException bootConfigValidationFailed(@Param ModelNode failureDescription);
```

Comment:
I'm not sure (and @jamezp will know for sure) but if you annotate the parameter with @Param, aren't we losing the failureDescription from the error message?

Reply:
Using `@Param` will use the `OperationFailedException(String, ModelNode)` constructor. Essentially it would be something like `new OperationFailedException("Boot configuration validation failed", failureDescription)`.